docs/pages/application-access/guides/api-access.mdx

```diff
@@ -70,7 +69,9 @@ target application's API through Teleport App Access.
   type="note"
   title="CA and Key Pair Files"
 >
-  Note the paths to the CA certificate and your user's certificate/key pair in the command - `curl` will use a client certificate to authenticate with Teleport.
+  Note the paths to your user's certificate/key pair in the command - `curl` will use a client certificate to authenticate with Teleport.
+  
+The Teleport Proxy Service is usually configured with a wildcard certificate issued by a public certificate authority such as Let's Encrypt. If your Teleport Proxy Service has been configured to use a self-signed certificate instead, you will need to include it in your `curl` command using `--cacert <path>`.
```

Comment:
I think we should surround this paragraph with a `<ScopedBlock scope={["oss", "enterprise"]}` to avoid misleading Cloud users.